docs/topics/azure-stack.md

```diff
@@ -346,6 +345,28 @@ The list below includes the addons currently unsupported on Azure Stack Hub:
 * Cluster Autoscaler
 * KeyVault Flex Volume
 * SMB Flex Volume
+* Azure Monitor for containers
+
+### Azure Disk CSI Driver 1.30+ not supported on Azure Stack Hub
+
+Azure Disk CSI Driver version 1.30 and later are not supported on Azure Stack Hub due to a dependency on a library that is not currently available on Azure Stack Hub. A fix is in progress and will be provided in a future AKS Engine release.
+
+**Current Recommendation:** Use the compatible Azure Disk CSI Driver version specified in the [Azure Disk CSI driver version mapping table](#azure-disk-csi-driver-version-mapping) that corresponds to your cluster's Kubernetes version.
+
+**If you have already installed Azure Disk CSI Driver 1.30+:** You need to uninstall the incompatible version and reinstall a supported version. Follow the steps below:
+
+**Uninstall Azure Disk CSI Driver:**
+
+* **If installed via Helm:**
+  ```bash
+  helm uninstall azuredisk-csi-driver --namespace kube-system
+  helm repo remove azuredisk-csi-driver
+  ```
+
+* **If installed via AKS Engine addon:**
+  1. Delete the `azuredisk-csi-driver-deployment.yaml` file from the `/etc/kubernetes/addons` directory on each master node to prevent auto-recreation by the Kubernetes addon manager
```

Comment:
Customer need to back up a copy of azuredisk-csi-driver-deployment.yaml, because it is required in next step.